File: EasyMarkets/app/src/main/java/merryweather/com/easymarkets/di/scopes/PerActivity.java
package merryweather.com.easymarkets.di.scopes;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Scope;

/**
 * Created by S on 17.05.2018.
 */

@Scope
@Retention(RetentionPolicy.RUNTIME)
public @interface PerActivity {
}

File: Tech45/app/src/main/java/merryweather/com/tech45/di/scopes/PerActivity.kt
package merryweather.com.tech45.di.scopes

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

import javax.inject.Scope

/**
 * Created by S on 17.05.2018.
 */

@Scope
@Retention(RetentionPolicy.RUNTIME)
annotation class PerActivity